build/make/build.mk

```diff
@@ -10,9 +12,9 @@ CUSTOM_GO_MOUNT?=-v /tmp:/tmp
 GO_BUILD_FLAGS?=-mod=vendor -a -tags netgo $(LDFLAGS) -installsuffix cgo -o $(BINARY)
 
 .PHONY: compile
-compile: $(BINARY)
+compile: $(BINARY) ## Compile the go program locally via Docker
```

Comment:
I think it is safe to say that we don't start compile actions on other, remote computers. I would suggest to remove `locally`